projects/wave-core-new/src/lib/operators/parameter-options/type-parameter-options-factory.service.ts

```diff
@@ -0,0 +1,20 @@
+import {OperatorTypeParameterOptions, OperatorTypeParameterOptionsConfig} from '../operator-type-parameter-options.model';
+// import {GdalSourceParameterOptions, GdalSourceParameterOptionsConfig} from './gdal-source-parameter-options.model';
+// import {GdalSourceType} from '../types/gdal-source-type.model';
+
+/**
+ * A simple factory for de-serializing operator parameter options.
+ */
+export abstract class OperatorTypeParameterOptionsFactory {
+    /**
+     * Create operator type from serialized data.
+     */
+    static fromDict(dict: OperatorTypeParameterOptionsConfig): OperatorTypeParameterOptions {
+        switch (dict.operatorType) {
+            // case GdalSourceType.TYPE:
+            //     return GdalSourceParameterOptions.fromDict(dict as GdalSourceParameterOptionsConfig);
+            default:
+                throw Error('There is not ParameterOptions factory method defined for the "' + dict.operatorType + '" type.');
+        }
+    }
+}
```

Comment:
Do we need this? I guess parameter options have to be real parametrized workflows in geo engine.

Reply:
We have no parametrized workflow yet

Reply:
Yeah, but so we can't load this because we don't store it, so -> I vote for removal

Reply:
a05473d8cac2769f869b74539507f8e6eb972f27